mirgecom/limiter.py

```diff
@@ -0,0 +1,108 @@
+"""
+:mod:`mirgecom.limiter` is for limiters and limiter-related constructs.
+
+Field limiter functions
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: bound_preserving_limiter
+.. autofunction:: cell_characteristic_size
+
+"""
+
+__copyright__ = """
+Copyright (C) 2022 University of Illinois Board of Trustees
+"""
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+from grudge.discretization import DiscretizationCollection
+import grudge.op as op
+
+
+def cell_characteristic_size(actx, dcoll: DiscretizationCollection):
```

Comment:
Any reason not to call this "cell_volume"?

Reply:
"Volume" reminds me of 3D.. So I wanted to use a more generic name for 2D as well.

Reply:
I am not fixing to hold merging up over this, but fwiw, I find `volume` to be sufficiently descriptive regardless of dimension, where `characteristic_size` leaves me wondering what this actually is; so I have to go look.   Luckily, in this case, it is an easily-understood one-liner.